packages/frontend/app/components/app/sidebar/chats/list.hbs

```diff
@@ -19,6 +19,19 @@
         <span class='icon'>{{fa-icon 'plus'}}</span>
       </a>
     </div>
+  {{else if (eq chat 'add-channel')}}
+    <div data-test-sidebar-channels-header class='menu-label'>
+      {{t 'ui.sidebar.channels.title'}}
+
+      <a
+        data-test-add-channel
+        class='button button-xs'
+        {{!-- TODO: link to creating channels --}} href={{url-for 'add-friend'}}
```

Comment:
actually, is all this pending the text commands PR?

Reply:
The text commands PR was meant as an intermediate step to allow us to get the underlying data moving/constructing right. As of now, all of these TODOs rely on some underlying method or construction that hasn't been implemented yet. We're going to have a discussion on what we want to do with that tomorrow, either before or during our meeting.